osu.Server.BeatmapSubmission/DatabaseOperationExtensions.cs

```diff
@@ -198,6 +198,18 @@ await db.ExecuteAsync(
                 transaction);
         }
 
+        public static async Task UpdateBeatmapCountForSet(this MySqlConnection db, uint beatmapSetId, uint beatmapCount, MySqlTransaction? transaction = null)
+        {
+            await db.ExecuteAsync(
+                "UPDATE IGNORE `osu_beatmapsets` SET `versions_available` = @versions_available WHERE `beatmapset_id` = @beatmapset_id",
```

Comment:
Is the `IGNORE` keyword here for a specific reason?

Reply:
...the only reason it is here is because it was there [in the old code too](https://github.com/peppy/osu-web-10/blob/466291788ab74605ffd100a24a939c2dbd10ef4a/www/web/osu-osz2-bmsubmit-getid.php#L193) :sweat_smile:

Reply:
I think we can just remove this.